news/2023/2023-08-05-skinning-contest-icons-of-an-era-results.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Skinning Contest: Icons of an Era - Results"
+date: 2023-08-05 00:00:00 +0000
+---
+
+Tune in to find out the results for the latest skinning contest, and snag yourself some amazing skins centred around the past 15 years of osu!
+
+![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/banner.jpg)
+
+The community has spoken with **3792** players casting a total of **9981** votes across **16** different top-of-the-notch skins, and out of all entries, the following 3 entries have climbed their way to the podium:
+
+## 3rd Place - QUAVER: DIVIDE BY 8
+
+**1022** quaver retry spammers decided [thetasigma](https://osu.ppy.sh/users/6234482)'s [QUAVER: DIVIDE BY 8](https://osu.ppy.sh/community/forums/topics/1792739) was worthy of their votes, netting themselves the **3rd** place!
+
+Based on [Sotarks](https://osu.ppy.sh/users/4452992)' [quaver](https://osu.ppy.sh/beatmapsets/873811#osu/1859185) beatmap set, this skin fully embodies the set's background and incorporates the universe theme across the entire skin. The song selection screen is beautifully done with various planets and stars floating about, and even [Stardust](https://vocaloid.fandom.com/wiki/Xingchen) from the original background makes an appearance across several gameplay screens. Congratulations to thetasigma for achieving 3rd place!
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/dVbEeRi0y_A" frameborder="0" allowfullscreen></iframe>
+</div>
+
+![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/3rd_badge@2x.png)
+
+## 2nd Place - Everlasting Moment
+
+Time stood still as **1058** voters cast their votes, resulting in a very close battle between 3rd and 2nd place. In the end, [Chromasia](https://osu.ppy.sh/users/7306251) was able to win the battle and achieve **2nd** place with their entry!
+
+[Everlasting Moment](https://osu.ppy.sh/community/forums/topics/1792308) keeps up with the universe theme, while providing a unique flair based off of the background art used in several beatmap sets of [Release Hallucination](https://osu.ppy.sh/beatmaps/artists/194)'s Chronostasis. The colour palette is in line with the original art, and that makes this skin very pleasing to use! Congratulations to Chromasia on their 2nd place!
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/MZHOHYw1ysw" frameborder="0" allowfullscreen></iframe>
+</div>
+
+![](/wiki/shared/news/2023-08-05-skinning-contest-icons-of-an-era-results/2nd_badge@2x.png)
+
+## Winner - FOOL MOON NIGHT
+
+[Spoo](https://osu.ppy.sh/users/11805037) was not a *fool* when they chose their theme (which somehow again includes a universe/space theme), because this skin stands head and shoulders above all other entries with a mind-blowing **1303** votes!
```

Comment:
```suggestion
[Spoo](https://osu.ppy.sh/users/11805037) was not a *fool* when they chose their theme (which somehow again involves the universe or space), because this skin stands head and shoulders above all other entries with a mind-blowing **1303** votes!
```